infrastructure/http/DefaultsExtractor.go

```diff
@@ -0,0 +1,53 @@
+package http
+
+import nethttp "net/http"
+
+type defaultsExtractor struct {
+}
+
+func (self defaultsExtractor) Extract(input map[string]interface{}) HttpActionState {
```

Comment:
Is it possible to wrap the `map[string]interface{}` in a type declaration for ease of readability?